### PR TITLE
Fix build without Xwayland support

### PR DESCRIPTION
**Pull request by Hummer12007**
separate options for X11 backend and Xwayland support

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -23,12 +23,15 @@ include(CTest)
 include(test)
 
 # Options
-OPTION(WLC_X11_SUPPORT "Build X11 backend and XWayland support" ON)
+OPTION(WLC_X11_BACKEND_SUPPORT "Build X11 backend" OFF)
+OPTION(WLC_XWAYLAND_SUPPORT "Support XWayland applications" OFF)
```
Comment (SirCmpwn) [inline CMakeLists.txt]: Default both to ON please

Comment (Hummer12007) [inline CMakeLists.txt]: Aaah, I thought OFF was the default (probably edited it myself while testing).
Thanks!

Comment (Hummer12007) [inline CMakeLists.txt]: Done!

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**